src/lib/builders/calendar/create.ts

```diff
@@ -514,20 +514,48 @@ export function createCalendar<
 	 * Updates the displayed months based on changes in the placeholder value,
 	 * which determines the months to show in the calendar.
 	 */
-	effect(
-		[placeholder, weekStartsOn, locale, fixedWeeks, numberOfMonths],
-		([$placeholder, $weekStartsOn, $locale, $fixedWeeks, $numberOfMonths]) => {
-			if (!isBrowser || !$placeholder) return;
+	effect([placeholder, weekStartsOn, locale, fixedWeeks, numberOfMonths], ([$placeholder]) => {
```

Comment:
```suggestion
	effect([placeholder], ([$placeholder]) => {
```

Reply:
We need to remove these so this only fires when the `placeholder` changes and we aren't doubling with the following effect.

Reply:
Duh I'm stupid lol...I removed initially, then CMD+Z to copy and forgot to re-remove it lol